eslint.config.mjs

```diff
@@ -10,7 +10,7 @@ import tseslint from 'typescript-eslint'
 
 export default tseslint.config(
   {
-    ignores: ['**/dist/', 'examples/**'],
+    ignores: ['dist/**', 'examples/**'],
```

Comment:
Actually, I don't think `**` is necessary.
```suggestion
    ignores: ['dist/', 'examples/'],
```

Reply:
@dai-shi That sounds good! 👍